Ionic_Example/e2e/src/app.e2e-spec.ts

```diff
@@ -0,0 +1,14 @@
+import { AppPage } from './app.po';
+
+describe('new App', () => {
+  let page: AppPage;
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+  it('should display welcome message', () => {
+    page.navigateTo();
+    expect(page.getPageTitle()).toContain('Tab One');
```

Comment:
Does this test still pass? I don't think we still have a tab with the text "Tab One"

Reply:
No in its state above it does not pass. Updated to match changes.